SU-SWS/stanford-jumpstart-deployer
# SOE-1579: update news and jumpstart_engineering

# READY
- Built and tested with latest Behat tests.

# Summary
- Updates the deployer to use the deprecated credits in favor of news caption/credits

# Needed By (Date)
- N/A

# Urgency
-  Needed before we roll the next PICAL site

# Steps to Test

1. Build a jse production site
2. Verify that the image credits field does not display when editing  a news item

# Affected Projects or Products
- None. Changes in this PR should be for JSE production sites only.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1684
- https://github.com/SU-SWS/stanford_news/releases/tag/7.x-3.6%2B23-dev
- https://github.com/SU-SOE/stanford_jumpstart_engineering/releases/tag/7.x-5.4%2B12-dev
- @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
